src/core/src/service_interfaces/LifecycleManagerAzure.py

```diff
@@ -0,0 +1,124 @@
+# Copyright 2020 Microsoft Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Requires Python 2.7+
+
+import json
+import os
+import shutil
+import time
+from core.src.bootstrap.Constants import Constants
+from core.src.service_interfaces.LifecycleManager import LifecycleManager
+
+class LifecycleManagerAzure(LifecycleManager):
```

Comment:
Please rename the corresponding test class of tests for this class (and edit test files to point to the right files). Please run all tests to ensure nothing has regressed, and address as appropriate.